clients/dashboards/dashboards.go

```diff
@@ -0,0 +1,78 @@
+package dashboards
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/influxdata/influx-cli/v2/api"
+	"github.com/influxdata/influx-cli/v2/clients"
+)
+
+type Client struct {
+	clients.CLI
+	api.DashboardsApi
+	api.OrganizationsApi
+}
+
+type Params struct {
+	clients.OrgParams
+	Ids []string
+}
+
+func (c Client) List(ctx context.Context, params *Params) error {
+	if !params.OrgID.Valid() && params.OrgName == "" && len(params.Ids) == 0 {
+		return fmt.Errorf("at least one of org, org-id, or id must be provided")
+	}
+
+	var req api.ApiGetOrgsRequest
```

Comment:
Ah, miscommunication. This logic is good but it can/should be running directly against the `GetDashboards` request below. I don't think there's any need to use the `GetOrgs` API in this method.